test/functional/hooks_functional.js

```diff
@@ -239,7 +263,7 @@ exports.mochaHooks = {
                 );
                 view.controls.lookAtCoordinate(init, false);
                 view.notifyChange();
-            } else if (view.isPlanarView) {
+            } else if (view && view.isPlanarView) {
                 // TODO: make the controls accessible from PlanarView before doing
```

Comment:
same remark here regarding the controls

Reply:
View.controls is not yet refactored between all the different type of Views. 
Thes part will need to be done before we can really cahnge the TODOs in here.